### DISCUSSION TITLE
make rest api calls in a sequence

### DISCUSSION THREAD

**Discussion by khyberkanwar** [Budibase/budibase#9189] [Help]
Hi,
I have two rest apis that I need to call in a sequence.  I need to extract info out of the result from the first api call and use that value as a parameter to the second call.

How would I capture the info from first api call and pass it to second api?

I am new to budibase and tried few things but have not succeeded.

Thanks.

**Comment by melohagan** [comment]
Hey @khyberkanwar 

You can do that by chaining **Execute Query** actions: https://docs.budibase.com/docs/chaining-actions#action-referencing
